.github/workflows/vale_linting.yml

```
@@ -0,0 +1,20 @@
+name: Vale linting
+on:
+  pull_request:
+
+jobs:
+  prose:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Vale
+      uses: errata-ai/vale-action@reviewdog
+      with:
+        version: 2.20.2
+        files: docs
```

As we're just in the process of adding i18n support for this repository, we should only check the files in the English locale (at least for starting out)

```suggestion
        files: docs/en
```

---

Or whatever other way we have for only including this directory of course